backend/app/controllers/spree/admin/reports_controller.rb

```diff
@@ -68,6 +48,29 @@ def sales_total
       end
 
       @@available_reports = {}
+
+      private
+
+      def search_params
+        params.fetch(:q, {}).tap do |q|
+          q[:completed_at_gt] = adjust_start_date q[:completed_at_gt]
+          q[:completed_at_lt] = adjust_end_date(q[:completed_at_lt]) if q[:completed_at_lt].present?
+          q[:s] ||= 'completed_at desc'
+        end
+      end
+
+      def adjust_start_date(string_date = nil)
+        return Time.current.beginning_of_month unless string_date
```

Comment:
The previous code checked the equivalent of `string_date.blank?`, should we still be doing that (I suspect yes since we may get an empty string from the user)?